material-ui/app/root.tsx

```diff
@@ -0,0 +1,67 @@
+import { PropsWithChildren, useContext } from "react";
+import type { MetaFunction } from "@remix-run/node";
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+import { withEmotionCache } from "@emotion/react";
+import { unstable_useEnhancedEffect as useEnhancedEffect } from "@mui/material";
+import "@fontsource/roboto/300.css";
+import "@fontsource/roboto/400.css";
+import "@fontsource/roboto/500.css";
+import "@fontsource/roboto/700.css";
```

Comment:
As you can see in our [styling guide](https://remix.run/guides/styling), you should import these via a [route module `links` export](https://remix.run/api/conventions#links)